recipes/spix/all/conanfile.py

```diff
@@ -91,7 +98,7 @@ def generate(self):
 
     def _patch_sources(self):
         apply_conandata_patches(self)
-        if Version(self.deps_cpp_info["qt"].version).major == 6:
+        if self.version == "0.4" and Version(self.deps_cpp_info["qt"].version).major == 6:
```

Comment:
deps cpp infor does not exist in 2.0 so this PR will need more short very soon :)

Reply:
Fixed